package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-api",
-  "version": "0.15.1",
+  "version": "0.16.0",
```

Comment:
This needs a similar change at: https://github.com/UKHomeOffice/lev-api/blob/master/.drone.yml#L4

Reply:
done

Reply:
Did you push your changes?

Reply:
I have now